pkg/platform/common/infracommon/dns.go

```diff
@@ -177,6 +195,16 @@ func (c *CommonPlatform) DeleteAppDNS(ctx context.Context, client ssh.Client, ku
 		if err != nil {
 			return err
 		}
+		if lbAPI != nil {
+			lbKey := edgeproto.LoadBalancerKey{
+				Name:      sn,
+				Namespace: namespace,
+			}
+			err := lbAPI.DeleteLoadBalancer(ctx, appInst.CloudletKey, appInst.ClusterKey, lbKey)
+			if err != nil {
```

Comment:
We probably need to continue here instead of reaturn - we still want to clean up dns records(line 194) for all services.